ang/civicase/case/list/directives/case-list-table-first-column-header.html

```diff
@@ -2,7 +2,7 @@
   ng-show="!viewingCase && selectedCases && bulkAllowed && !item.lock"
   class="pull-left"
   show-checkboxes="$parent.showCheckboxes"
-  is-select-all-available="isSelectAllAvailable"
+  is-select-all-available="true"
```

Comment:
why pass this parameter if it was removed from the controller?

Also make sure to remove it from everywhere the `civicase-bulk-actions-message` is implemented.

Reply:
This attribute is not necessary anymore, removed from the directive as well.